selfdrive/car/toyota/carstate.py

```diff
@@ -60,6 +55,33 @@ def get_can_parser(CP):
     ("EPS_STATUS", 25),
   ]
 
+  if not CP.carFingerprint == CAR.LEXUS_ISH:
+    signals += [
+      ("MAIN_ON", "PCM_CRUISE_2", 0),
+      ("SET_SPEED", "PCM_CRUISE_2", 0),
+      ("LOW_SPEED_LOCKOUT", "PCM_CRUISE_2", 0),
+      ("IPAS_STATE", "EPS_STATUS", 1),
+      ("AUTO_HIGH_BEAM", "LIGHT_STALK", 0),
+    ]
+
+  if CP.carFingerprint == CAR.LEXUS_ISH:
+    checks = [
+      ("BRAKE_MODULE", 50),
+      ("GAS_PEDAL", 50),
+      ("WHEEL_SPEEDS", 80),
+      ("STEER_ANGLE_SENSOR", 80),
+      ("PCM_CRUISE", 33),
+      ("PCM_CRUISE_ISH", 1),
+      ("STEER_TORQUE_SENSOR", 50),
+      ("EPS_STATUS", 25),
+    ]
+
+    signals += [
+      ("MAIN_ON", "PCM_CRUISE_ISH", 0),
+      ("SET_SPEED", "PCM_CRUISE_ISH", 0),
+      ("AUTO_HIGH_BEAM", "LIGHT_STALK_ISH", 0),
+    ]
+
```

Comment:
Can you clean this code up? Something like this:
```suggestion
  checks = [
    ("WHEEL_SPEEDS", 80),
    ("STEER_ANGLE_SENSOR", 80),
    ("PCM_CRUISE", 33),
    ("STEER_TORQUE_SENSOR", 50),
    ("EPS_STATUS", 25),
  ]

  if CP.carFingerprint == CAR.LEXUS_ISH:
    checks += [
      ("BRAKE_MODULE", 50),
      ("GAS_PEDAL", 50),
      ("PCM_CRUISE_ISH", 1),
    ]
    signals += [
      ("MAIN_ON", "PCM_CRUISE_2", 0),
      ("SET_SPEED", "PCM_CRUISE_2", 0),
      ("LOW_SPEED_LOCKOUT", "PCM_CRUISE_2", 0),
      ("IPAS_STATE", "EPS_STATUS", 1),
      ("AUTO_HIGH_BEAM", "LIGHT_STALK", 0),
    ]

  else:
    checks += [
      ("BRAKE_MODULE", 40),
      ("GAS_PEDAL", 33),
      ("PCM_CRUISE_2", 33),
    ]

     signals += [
      ("MAIN_ON", "PCM_CRUISE_ISH", 0),
      ("SET_SPEED", "PCM_CRUISE_ISH", 0),
      ("AUTO_HIGH_BEAM", "LIGHT_STALK_ISH", 0),
    ]
```

Reply:
@legonigel no worries, sorted